pandas/tests/tslibs/test_parsing.py

```diff
@@ -24,7 +24,8 @@ def test_parse_time_string():
 
 def test_parse_time_string_invalid_type():
     # Raise on invalid input, don't just return it
-    with pytest.raises(TypeError):
+    msg = "Invalid time string provided, unable to parse"
+    with pytest.raises(TypeError, match=msg):
         parse_time_string((4, 5))
```

Comment:
looks like the message raised here includes the arg beinga  tuple

Reply:
Sorted this out now.